examples/github-complete/main.tf

```diff
@@ -37,6 +37,9 @@ module "atlantis" {
   private_subnets = ["10.20.1.0/24", "10.20.2.0/24", "10.20.3.0/24"]
   public_subnets  = ["10.20.101.0/24", "10.20.102.0/24", "10.20.103.0/24"]
 
+  # EFS
+  enable_ephemeral_storage = true
```

Comment:
It is enough to have this argument in one example. I have removed the copied one.